apps/web/src/components/Notifications/notifications/SomeoneAdmiredYourPost.tsx

```diff
@@ -40,21 +39,25 @@ export default function SomeoneAdmiredYourPost({ notificationRef, onClose }: Pro
     `,
     notificationRef
   );
-  const reportError = useReportError();
+
   const notificationCount = notification.count ?? 1;
 
   const firstAdmirer = notification.admirers?.edges?.[0]?.node;
-  const token = notification.post?.tokens?.[0];
-  const previewUrlSet = useMemo(() => {
-    if (!token) return null;
-    return getVideoOrImageUrlForNftPreview({ tokenRef: token });
-  }, [token]);
 
   if (!firstAdmirer) {
-    reportError(`SomeoneAdmiredYourPostNotification id:${notification.dbid} was missing admirer`);
-    return null;
+    throw new ErrorWithSentryMetadata('SomeoneAdmiredYourPostNotification was missing admirer', {
+      id: notification.dbid,
+    });
```

Comment:
note: if a component should error, we should throw an error rather than call `reportError` directly in the render function; some parent component should catch the error in a boundary

in this case, a `ReportingErrorBoundary` was wrapped around the `Notification` component